ditaa-ditaa.rb

```diff
@@ -59,8 +67,25 @@ def initialize site, options = { }
 			@options.keep_if { |key, _| self.class::Options.include? key.to_s }
 		end
 		
+		def imgsize
+			@size ||= FastImage.size destination @site.config["destination"]
+			{
+				"width" => @size[0],
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.<br>Tab detected.